### PR TITLE
marketing deploy remote and key fixes

### PR DESCRIPTION
**Pull request by macdiesel**
More fixes for marketing.

### CODE REVIEW

File: edxpipelines/pipelines/deploy_marketing_site.py [hunk 1]
```diff
@@ -80,6 +80,8 @@ def install_pipelines(configurator, config, env_configs):  # pylint: disable=unu
             """\
             echo -n "release-$(date +%Y-%m-%d-%H.%M)" > ../target/{new_tag}.txt &&
             TAG_NAME=$(cat ../target/{new_tag}.txt) &&
+            /usr/bin/git config user.email "admin@edx.org" &&
+            /usr/bin/git config user.name "edx-secure" &&
```
Comment (doctoryes) [inline edxpipelines/pipelines/deploy_marketing_site.py]: Are these the config values we use for other git commands? If so, seems strange to set them here and not globally. If not, will this config-tweaking screw up other jobs?

File: edxpipelines/pipelines/rollback_prod_marketing_site.py [hunk 2]
```diff
@@ -73,7 +73,10 @@ def install_pipelines(configurator, config, env_configs):  # pylint: disable=unu
     tasks.generate_package_install(clear_prod_caches_job, 'tubular')
     clear_prod_caches_job.add_task(
         tasks.bash_task(
-            'cp {ecom_secure}/acquia/acquia_github_key.pem {edx_mktg}/docroot/',
+             """
```
Comment (doctoryes) [inline edxpipelines/pipelines/rollback_prod_marketing_site.py]: Not lined up.

File: edxpipelines/pipelines/deploy_marketing_site.py (hunk 1 above)
Comment (macdiesel) [inline edxpipelines/pipelines/deploy_marketing_site.py]: yeah, I agree it's strange, I tried removing them earlier only to have the command fail. Lets put them back in place for now and we can deal with some of this marketing pipeline oddness when @Dillon-Dumesnil returns!

Comment (macdiesel) [inline edxpipelines/pipelines/deploy_marketing_site.py]: Also to this point this config tweaking hasn't messed up other jobs. Not to say that there are not side effects, but none that we have noticed yet.

Comment (cpennington) [inline edxpipelines/pipelines/deploy_marketing_site.py]: We use it when we're merging things, but in those cases, we set them via environment variable.

**Pull request merged**